src/components/Navbar/Navbar.jsx

```diff
@@ -21,7 +24,7 @@ export const Navbar = () => {
             <LinkC as={Link} to={paths.profile}>
               Profile 
             </LinkC>
-            <LinkC as={Link} to={paths.signup}>Sign up</LinkC>
+           {navbar && <LinkC as={Link} to={paths.signup}>Sign up</LinkC>}
```

Comment:
I'm sorry but same feedback as before. You should aim to only have relevant changes to the task in a PR.

Reply:
I am sorry I made mistake. I deleted those changes but I made a mistake and push it because I use 5 branches at the same time.

Reply:
You've reverted signup.hook 👍 can you revert this change as well?